fp-proj/src/app/Modules/products/products.module.ts

```diff
@@ -0,0 +1,18 @@
+import { NgModule } from '@angular/core';
```

Comment:
niepotrzebny jest ten katalog Modules. mozna poporstudodawc kolejne moduly w osobnych katalogach. jesli nie ma za duzo komponentow w glownym module to struktura powinna byc dalej jasna  czysta